### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by dav3r**
This PR adds some more useful info to the README.  It also includes some whitespace changes to make pre-commit happy.

### CODE REVIEW

File: terraform/acl_rules.tf [hunk 1]
```diff
@@ -2,24 +2,24 @@
...
 resource "aws_network_acl_rule" "ingress_from_anywhere_via_any_port" {
   network_acl_id = "${aws_network_acl.vuln_acl.id}"
-  egress = false
-  protocol = "-1"
-  rule_number = 100
-  rule_action = "allow"
-  cidr_block = "0.0.0.0/0"
-  from_port = 0
-  to_port = 0
+  egress         = false
```
Comment (jsf9k) [inline terraform/acl_rules.tf:5]: I detest this style, but Terraform does appear to advocate it.

File: README.md [hunk 2]
```diff
@@ -5,6 +5,118 @@
 This project allows you to create intentionally-vulnerable virtual
 machines for testing purposes.
 
+## Requirements ##
+
+* [AWS CLI access
+  configured](
+  https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+  on your system
+
```
Comment (jsf9k) [inline README.md]: Why the blank lines between bullets?

**Review by jsf9k — APPROVED** [review]
LGTM, but see my comment about unnecessary blank lines between bullets at the top of the README.

File: README.md (hunk 2 above)
Comment (dav3r) [reply README.md]: Thanks- I was fighting with a linter message about blank lines between lists, but it turned out to be an issue with not indenting the subsequent lines of the list items.  Anyway, I got rid of those blank lines in 8673edc.

File: terraform/acl_rules.tf (hunk 1 above)
Comment (dav3r) [reply terraform/acl_rules.tf:5]: I don't love it, but since `terraform_fmt` takes care of it automatically, I don't really care that much.

**Pull request merged**